internal/puppeteer/config.go

```diff
@@ -43,6 +44,7 @@ type Suite struct {
 	Browser   string            `yaml:"browser,omitempty" json:"browser"`
 	TestMatch []string          `yaml:"testMatch,omitempty" json:"testMatch"`
 	Env       map[string]string `yaml:"env,omitempty" json:"env"`
+	Args      []string          `yaml:"args,omitempty" json:"args"`
```

Comment:
Would be good to add more info to this PR. I had to look up the ticket to figure what this was about ⚠️ 

Now to the actual question:
Any example on how you think this ought to be used by the user? Since some args have key-value pairs.

Like this?
```yaml
args:
  - --disable-features=site-per-process
```

Reply:
yep. like
```
args:
  - "--disable-features=site-per-process"
```
I checked some browser args. Some are key-value pair and some are only flags. So I just choose `[]string` here.

Reply:
Gotcha!
One more thing: It's unclear that these args are browser args. Would be good to rename it as such, since these are not jest args or puppeteer args or others, but specifically for browsers.

Reply:
I see!